src/apps/backend/modules/share-task-request/rest-api/share-task-request-router.ts

```diff
@@ -0,0 +1,15 @@
+import { accessAuthMiddleware } from '../../access-token';
+import { ApplicationRouter } from '../../application';
+import { ShareTaskRequestController } from './share-task-request-controller';
+
+export default class ShareTaskRequestRouter extends ApplicationRouter {
+  configure(): void {
+    const { router } = this;
+    const controller = new ShareTaskRequestController();
+
+    router.use(accessAuthMiddleware);
+
+    router.post('/', controller.createSharedTask);
+    router.get('/', controller.getSharedTasks);
```

Comment:
This GET API is not required as we already have `/tasks` API to get all tasks so that can be used to get shared tasks also. You can pass query params as `sharedTask = true`.